test/Summary.spec.js

```diff
@@ -0,0 +1,51 @@
+describe('The summary', function() {
+    let page;
+
+    before(async function() {
+        page = await browser.newPage();
+        await page.goto('http://localhost:8080');
+    });
+
+    after(async function() {
+        await page.close();
+    });
+
+    describe('initially', function() {
+
+        it("should not exist", async function() {
+            const summariescontainer = await page.$(SEL_SIDEBAR_CONTAINER);
+            expect(summariescontainer).to.be.null;
+        });
+    });
+
+    describe('after submission', function() {
+
+        before(async function() {
+            await page.type(SEL_TEXTAREA, TEXT);
+
+            const button = await page.$(SEL_SUBMIT);
+            await button.click();
+        });
+
+        it('should appear after submission', async function() {
+            const summariescontainer = await page.$(SEL_SIDEBAR_CONTAINER);
+            expect(summariescontainer).to.exist;
+        });
+
+        it('should have correct number of summaries', async function() {
+            await page.waitFor(SEL_ISSUE_P);
+            expect(await page.$$(SEL_ISSUE_P)).to.have.lengthOf(2);
+        });
+
+        it('should be highlighted on mouseover', async function() {
+            await page.hover(SEL_ISSUE);
+
+            const issueStyle = await page.evaluate(issue => {
+                const sel = document.querySelector(issue);
+                return JSON.parse(JSON.stringify(getComputedStyle(sel)));
+            }, SEL_ISSUE);
+
+            expect(issueStyle.background).to.include('rgb(98, 176, 240)');
```

Comment:
You should be testing for a style class, not a specific color. If you can't figure out how to do that, a comment of explanation is needed.

Reply:
Understood. I was able to do that quite easily in 3 lines. It 'hovers' over the 'issue' class and checks if 'issue:hover' class appears. If there is no hover event simulated, we get a timeout error.